pkg/controller/issueapi/metrics.go

```diff
@@ -117,25 +119,49 @@ func registerMetrics() (*Metrics, error) {
 		return nil, fmt.Errorf("stat view registration failure: %w", err)
 	}
 
-	mRealmCapacity := stats.Float64(MetricPrefix+"/realm_capacity", "Available capacity for issuing verification codes", stats.UnitDimensionless)
+	mRealmTokenRemaining := stats.Int64(MetricPrefix+"/realm_token_remaining", "Remaining number of verification codes", stats.UnitDimensionless)
 	if err := view.Register(&view.View{
-		Name:        MetricPrefix + "/realm_capacity_latest",
-		Description: "Latest realm capacity",
+		Name:        MetricPrefix + "/realm_token_remaining_latest",
```

Comment:
Should these be `_count` instead of `_latest`?

Reply:
I thought `_count` was for cases where aggregation is `Count`. But do you mean keep the suffix as `_count` regardless of aggregation type?

Reply:
nvm I think you're right